packages/Checkbox/CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [0.2.0] - 2020-01-14
+
 ### Removed
+
 - checkboxStates import which was broken [@oscarkwan](https://github.com/oscarkwan).
 
 ### Added
+
 - Checkbox.states instead of checkboxStates [@oscarkwan](https://github.com/oscarkwan).
+
+## [0.2.2] - 2020-01-31
+
+### Added
+
+- Added tabIndex prop which is forwarded to input [@tristanjasper](https://github.com/tristanjasper).
```

Comment:
Not strictly needed, according to the process we agreed on (only breaking changes need to be added to changelog), but this _is_ appreciated, thanks! 🙇